### PR TITLE
zh translation update

### PR DESCRIPTION
**Pull request by jocelyn220**

### CODE REVIEW

**Comment by BingoRUS** [comment]
@beginor as an initial contributor of the Chinese localization, could you please check these changes?

**Comment by beginor** [comment]
OK， I am coming！

File: js/localization/messages/zh.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "Back": "返回",
     "OK": "确定",
 
-    "dxCollectionWidget-noDataText": "没有要显示的数据",
+    "dxCollectionWidget-noDataText": "没有可显示的数据",
```
Comment (beginor) [inline js/localization/messages/zh.json]: for short, it should be "无数据"

File: js/localization/messages/zh.json [hunk 2]
```diff
@@ -53,18 +53,18 @@
...
     "dxFileUploader-Gb": "Gb",
     "dxFileUploader-upload": "上传",
-    "dxFileUploader-uploaded": "上传",
+    "dxFileUploader-uploaded": "已上传",
     "dxFileUploader-readyToUpload": "准备上传",
     "dxFileUploader-uploadFailedMessage": "上传失败",
-    "dxFileUploader-invalidFileExtension": "",
-    "dxFileUploader-invalidMaxFileSize": "",
-    "dxFileUploader-invalidMinFileSize": "",
+    "dxFileUploader-invalidFileExtension": "档案类型错误",
```
Comment (beginor) [inline js/localization/messages/zh.json]: "档案" should be "文件", "档案" is only used in traditional chinese 。

File: js/localization/messages/zh.json [hunk 3]
```diff
@@ -53,18 +53,18 @@
...
+    "dxFileUploader-invalidMaxFileSize": "档案过大",
```
Comment (beginor) [inline js/localization/messages/zh.json]: "档案" should be "文件", "档案" is only used in traditional chinese 。

File: js/localization/messages/zh.json [hunk 4]
```diff
@@ -53,18 +53,18 @@
...
+    "dxFileUploader-invalidMinFileSize": "档案过小",
```
Comment (beginor) [inline js/localization/messages/zh.json]: "档案" should be "文件", "档案" is only used in traditional chinese 。

File: js/localization/messages/zh.json [hunk 5]
```diff
@@ -78,19 +78,19 @@
 
     "dxDataGrid-columnChooserTitle": "列选择器",
     "dxDataGrid-columnChooserEmptyText": "在这里拖动一列隐藏它",
-    "dxDataGrid-groupContinuesMessage": "下一页",
-    "dxDataGrid-groupContinuedMessage": "上一页",
-    "dxDataGrid-groupHeaderText": "通过这列分组",
+    "dxDataGrid-groupContinuesMessage": "至下一页",
```
Comment (beginor) [inline js/localization/messages/zh.json]: I think it is no need to change.

File: js/localization/messages/zh.json [hunk 6]
```diff
@@ -78,19 +78,19 @@
...
+    "dxDataGrid-groupContinuedMessage": "回到上一页",
```
Comment (beginor) [inline js/localization/messages/zh.json]: I think it is no need to change.

File: js/localization/messages/zh.json [hunk 7]
```diff
@@ -78,19 +78,19 @@
...
+    "dxDataGrid-groupHeaderText": "通过该列分组",
     "dxDataGrid-ungroupHeaderText": "取消分组",
     "dxDataGrid-ungroupAllText": "取消所有分组",
     "dxDataGrid-editingEditRow": "编辑",
-    "dxDataGrid-editingSaveRowChanges": "保存",
+    "dxDataGrid-editingSaveRowChanges": "储存",
```
Comment (beginor) [inline js/localization/messages/zh.json]: should be "保存" , "存储" is usually used in traditional chinese .

File: js/localization/messages/zh.json [hunk 8]
```diff
@@ -264,24 +264,24 @@
     "dxFilterBuilder-filterOperationContains": "包含",
     "dxFilterBuilder-filterOperationNotContains": "不包含",
     "dxFilterBuilder-filterOperationEndsWith": "结束于",
-    "dxFilterBuilder-filterOperationIsBlank": "空",
-    "dxFilterBuilder-filterOperationIsNotBlank": "不为空",
+    "dxFilterBuilder-filterOperationIsBlank": "空白",
```
Comment (beginor) [inline js/localization/messages/zh.json]: should be "空“

File: js/localization/messages/zh.json [hunk 9]
```diff
@@ -264,24 +264,24 @@
...
+    "dxFilterBuilder-filterOperationIsNotBlank": "不为空白",
```
Comment (beginor) [inline js/localization/messages/zh.json]: should be “不为空”

File: js/localization/messages/zh.json [hunk 10]
```diff
@@ -264,24 +264,24 @@
...
-    "dxHtmlEditor-dialogLinkTextField": "!TODO!",
-    "dxHtmlEditor-dialogLinkTargetField": "!TODO!",
-    "dxHtmlEditor-dialogImageCaption": "!TODO!",
-    "dxHtmlEditor-dialogImageUrlField": "!TODO!",
-    "dxHtmlEditor-dialogImageAltField": "!TODO!",
-    "dxHtmlEditor-dialogImageWidthField": "!TODO!",
-    "dxHtmlEditor-dialogImageHeightField": "!TODO!",
-    "dxHtmlEditor-heading": "!TODO!",
-    "dxHtmlEditor-normalText": "!TODO!"
+    "dxHtmlEditor-dialogColorCaption": "变更字体颜色",
```
Comment (beginor) [inline js/localization/messages/zh.json]: should be “更改字体颜色”

File: js/localization/messages/zh.json [hunk 11]
```diff
@@ -264,24 +264,24 @@
...
+    "dxHtmlEditor-dialogBackgroundCaption": "变更背景颜色",
```
Comment (beginor) [inline js/localization/messages/zh.json]: should be "更改背景颜色"

File: js/localization/messages/zh.json [hunk 12]
```diff
@@ -264,24 +264,24 @@
...
+    "dxHtmlEditor-dialogLinkCaption": "添加连结",
```
Comment (beginor) [inline js/localization/messages/zh.json]: should be “添加链接”

File: js/localization/messages/zh.json [hunk 13]
```diff
@@ -264,24 +264,24 @@
...
+    "dxHtmlEditor-dialogLinkUrlField": "网址",
```
Comment (beginor) [inline js/localization/messages/zh.json]: ”链接地址“

File: js/localization/messages/zh.json [hunk 14]
```diff
@@ -264,24 +264,24 @@
...
+    "dxHtmlEditor-dialogLinkTextField": "文字",
```
Comment (beginor) [inline js/localization/messages/zh.json]: “链接文字”

File: js/localization/messages/zh.json [hunk 15]
```diff
@@ -264,24 +264,24 @@
...
+    "dxHtmlEditor-dialogLinkTargetField": "于新视窗中开启",
```
Comment (beginor) [inline js/localization/messages/zh.json]: “在新窗口中打开”

File: js/localization/messages/zh.json [hunk 16]
```diff
@@ -264,24 +264,24 @@
...
+    "dxHtmlEditor-dialogImageCaption": "添加影像",
```
Comment (beginor) [inline js/localization/messages/zh.json]: “添加图片”

File: js/localization/messages/zh.json [hunk 17]
```diff
@@ -264,24 +264,24 @@
...
+    "dxHtmlEditor-dialogImageUrlField": "网址",
```
Comment (beginor) [inline js/localization/messages/zh.json]: “图片地址”

File: js/localization/messages/zh.json [hunk 18]
```diff
@@ -264,24 +264,24 @@
...
+    "dxHtmlEditor-dialogImageAltField": "替代文字",
+    "dxHtmlEditor-dialogImageWidthField": "宽 (px)",
+    "dxHtmlEditor-dialogImageHeightField": "长 (px)",
+    "dxHtmlEditor-heading": "标题",
+    "dxHtmlEditor-normalText": "一般文字"
```
Comment (beginor) [inline js/localization/messages/zh.json]: ”正文“

**Review by beginor — CHANGES_REQUESTED** [review]
I have add some comment, please do a double check.

File: js/localization/messages/zh.json (hunk 1 above)
Comment (BingoRUS) [reply js/localization/messages/zh.json]: so which variant is more preferable - "没有要显示的数据", "没有可显示的数据" or "无数据"?

Comment (beginor) [reply js/localization/messages/zh.json]: The English version of `dxCollectionWidget-noDataText` is "No data to display", I think the best should be "没有要显示的数据" ， “无数据” is just for short

Comment (BingoRUS) [reply js/localization/messages/zh.json]: okay, thank you!

**Comment by BingoRUS** [comment]
@jocelyn220 I made some changes according to the initial contributor review and going to merge this PR tomorrow. Please let me know if you have any additions.

**Review by BingoRUS — APPROVED** [review]

**Pull request merged**